packages/driver/test/cypress/integration/commands/connectors_spec.coffee

```diff
@@ -752,21 +751,40 @@ describe "src/cy/commands/connectors", ->
 
           cy.wrap(obj).its("foo.bar.baz").should("eq", "baz")
 
-        it "throws when property does not exist on the subject", (done) ->
-          cy.on "fail", (err) =>
-            lastLog = @lastLog
+        it "does not throw when property does not exist on the subject", ->
+          cy.noop({}).its("foo")
 
-            expect(err.message).to.include "cy.its() errored because the property: 'foo' does not exist on your subject."
-            expect(lastLog.get("error").message).to.include(err.message)
+        it "retries when yielded undefined value", ->
+          obj = {}
+
+          obj.foo = undefined
+
+          setTimeout ()->
+            obj.foo = true
+          , 10
+
+          cy.wrap(obj).its('foo').should('eq', true)
+
+        it "can handle getter that throws", (done) ->
+          spy = cy.spy((err)=>
+            # throw new Error('sdf')
+            # expect(false).to.eq(true)
+            expect(err.message).to.eq('Timed out retrying: some getter error')
             done()
+            ).as('onFail')
+          cy.on 'fail', spy
+          obj = {}
+          Object.defineProperty obj, 'foo', {get: -> throw new Error('some getter error')}
+
+          cy.wrap(obj).its('foo')
+
 
-          cy.noop({}).its("foo")
 
         it "throws when reduced property does not exist on the subject", (done) ->
           cy.on "fail", (err) =>
             lastLog = @lastLog
 
-            expect(err.message).to.include "cy.its() errored because the property: 'baz' does not exist on your subject."
+            expect(err.message).to.include   "Timed out retrying: cy.its() errored because the property: 'baz' returned a 'undefined' value. You cannot access any properties such as 'fizz' on a 'undefined' value."
```

Comment:
I'm not sure if this error message always prints 'undefined', but if it does, the wording should be updated from "a 'undefined'" to "an 'undefined'"

Reply:
going to ignore this for now, since I didn't write the error message in this PR, and our error messages don't easily support conditional verbiage